srcpkgs/termdown/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'termdown'
+pkgname=termdown
+version=1.16.0
+revision=1
+archs=noarch
+build_style=python3-module
+pycompile_module="termdown"
+hostmakedepends="python3-setuptools"
+depends="python3 python3-dateutil python3-click python3-pyfiglet"
```

Comment:
You can remove `python3` since it's pulled in by the other python dependencies

Reply:
Did it